docs/references/failures.mdx

```diff
@@ -78,6 +78,7 @@ During conversion, the following Application Failure fields are set:
 - `non_retryable` is set to false.
 - `details` are left unset.
 - `cause` is a Failure converted from the error's `cause` property.
+- `next_retry_delay` is left unset.
```

Comment:
Kept the underscores here for consistency and since it's referring to server fields.  Otherwise it's consistently capitalized.